backend/graphql/internal/schema/README.txt

```diff
@@ -0,0 +1,4 @@
+1) Re-run "go run github.com/99designs/gqlgen" when you update the schema.
```

Comment:
would it possible to document the structure of the folder, when would one end up updating which file?

Reply:
ack

Reply:
Done